src/state/messages/convo/agent.ts

```diff
@@ -1031,7 +1031,10 @@ export class Convo {
            * `getItems` is only run in "active" status states, where
            * `this.sender` is defined
            */
-          sender: this.sender!,
+          sender: {
+            $type: 'chat.bsky.convo.defs#messageViewSender',
+            did: this.sender!.did,
+          },
```

Comment:
Here's an example of passing the full `ProfileView*` where only `did` was needed that was previously possible with the old loose types. This was only used to construct "pending" messages and never written to the user's repo, but it's good to be precise.